site/booster-2022.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: layouts/page.njk
+title: Booster 2022 abstracts
+description: To be or not to be, that is the question
+---
+
+## 30 min talk ideas
+
+### Building a healthy company culture
+
+When I rebuilt VIBBIO's product team from scratch in 2018, we made the conscious decision to embrace a fully remote setup. This gave us an advantage in recruiting but brought a significant change for the company.
+
+With a fully remote team, I could not get away with the all too pervasive management style of muddling through, I had to make deliberate choices about pratices and principles to foster the culture that I wanted to see.
+
+In this presentation, I'll talk about the challenges I faced, the lessons I learned, and give some insight into how good management principles can change companies for the better.
+
+### A primer on setting goals and how to measure progress
+
+We've probably all been there: someone hands us a KPI or any other kind of goal, and we just cannot make sense of it. Why this number, where did it come from, and why is this important now?
```

Comment:
I think that it would help the abstract if the three-letter-acronyms would be named before used. The Booster Conf crowd has many different background and I am not sure everybody is familiar with Key Performance Indicators and Objective and Key results.